src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -158,4 +158,14 @@ export class EcoverseService {
       throw e;
     }
   }
+
+  ///////////////////////////// Mutations ////////////////////////////////////
```

Comment:
I'd omit this comment. Although the method is used in a mutation, the service is providing business logic => this method can very well be consumed in a different way, e.g. rest service.